core/primitives/src/views.rs

```diff
@@ -274,6 +274,8 @@ pub enum QueryRequest {
         account_id: AccountId,
         #[serde(rename = "prefix_base64", with = "base64_format")]
         prefix: StoreKey,
+        #[serde(default, skip_serializing_if = "is_false")]
+        include_proof: bool,
```

Comment:
`remember_visited_nodes` works correctly.  If `include_proof` is false than `proof` field will be an empty vector.  The reason for the disclaimer is that I want to remove the field in the future but want to keep it now for compatibility.  Note that the `proof` fields were defined and sent long before we had proof functionality so I want to avoid situation where a client expects the field to be present (even though it historically has always been an empty array) and suddenly breaks when we stop sending the field if its empty.